### PR TITLE
feat(frontmatter): Document required vs. optional frontmatter

### PR DESCRIPTION
**Pull request by janpio**
See https://prisma-company.slack.com/archives/C5Z9TH6N9/p1686763363942739

### CODE REVIEW

File: content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx [hunk 1]
```diff
@@ -43,29 +48,35 @@ Note:
...
+### <inlinecode>preview</inlinecode>
 
 Accepts `true` or `false` (defaults to `false`).
 
 Adds a `preview` label to a page in the left-hand navigation.
 
-## <inlinecode>toc</inlinecode>
+### <inlinecode>toc</inlinecode>
 
 Enable or disable table of contents navigation on the page (defaults to `false`). For example:
```
Comment (janpio) [inline content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx]:
```suggestion
Enable or disable table of contents navigation on the page (defaults to `true`). For example:
```

**Review by meletj — COMMENTED** [review]
Overall LGTM. I'll let Nilu take a second look too. 
I think we could adjust this to use 'you' instead of 'we' . _If we wish, we can make the `metaTitle` more descriptive than the `title`._

**Comment by janpio** [comment]
Unfortunately seems that the frontmatter page was changed too much so there is now a conflict. This will need to be re-applied to the current state @nilubava, @carlagn, @meletj.

**Comment by nikolasburk** [comment]
Yeah, I tried to go through it in the web editor but wasn't able to properly resolve everything. Let's defer 🙏

**Comment by janpio** [comment]
Reordering is almost impossible to get right here in the GitHub UI. I think someone will need to look at the preview of this, resort the current state of the frontmatter page, and then add the same headlines as here.

**Pull request closed without merging**